nx/include/switch/services/capsa.h

```diff
@@ -18,31 +18,268 @@ void capsaExit(void);
 /// Gets the Service for caps:a.
 Service* capsaGetServiceSession(void);
 
+/// Gets the Service for IAlbumAccessorSession, only initialized after \ref capsaOpenAlbumMovieStream was used (unaffected by using \ref capsaCloseAlbumMovieStream).
+Service* capsaGetServiceSession_Accessor(void);
+
+/**
+ * @brief Gets the amount of files at a AlbumStorage.
+ * @param[in] storage \ref CapsAlbumStorage
+ * @param[out] count Amount of files.
+ */
 Result capsaGetAlbumFileCount(CapsAlbumStorage storage, u64* count);
+
+/**
+ * @brief Gets a listing of \ref CapsAlbumEntry, where the AlbumFile's storage matches the input one.
+ * @param[in] storage \ref CapsAlbumStorage
+ * @param[out] count Total output entries.
+ */
 Result capsaGetAlbumFileList(CapsAlbumStorage storage, u64* count, CapsAlbumEntry* buffer, u64 size);
+
+/**
+ * @brief Loads a file into the specified buffer.
+ * @param[in] file_id \ref CapsAlbumFileId
+ * @param[out] out_size Size of the AlbumFile.
+ * @param[out] workbuf File output buffer.
```

Comment:
Is this buffer cleared or not?(update comments depending on this) Same for the equivalent buffers elsewhere.

Reply:
Bad phrasing I'd say as this isn't a work buffer.